hack/nightly.sh

```diff
@@ -4,19 +4,6 @@ RELEASE_REPO=${RELEASE_REPO:-public.ecr.aws/z3c6l6z9/karpenter-nightly}
 RELEASE_VERSION=${RELEASE_VERSION:-$(date "${NIGHTLY_TAG_FMT}")}
 RELEASE_PLATFORM="--platform=linux/amd64,linux/arm64"
 
-if [ -z "$CLOUD_PROVIDER" ]; then
-    echo "CLOUD_PROVIDER environment variable is not set: 'export CLOUD_PROVIDER=aws'"
-    exit 1
-fi
-
-# TODO restore https://reproducible-builds.org/docs/source-date-epoch/
```

Comment:
Are you removing this because it's been completed?

Reply:
No those were copied from release.sh where there's cosign they are not used in here.